linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -279,6 +279,8 @@ fi
 /usr/lib/tmpfiles.d/%{name}.conf
 
 %changelog
+* Thu Aug 08 2022 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 8.0.345.0.0.1.adopt0
```

Comment:
@sxa Same log is needed for SUSE.

Reply:
This PR has already been merged - any further updates would need to be in a separate PR